integration-tests/src/tests/nearcore/node_cluster.rs

```diff
@@ -42,7 +42,7 @@ pub fn start_nodes(
             if i == 0 { first_node } else { open_port() },
             genesis.clone(),
         );
-        rpc_addrs.push(near_config.rpc_addr().unwrap().clone());
+        rpc_addrs.push(near_config.rpc_addr().unwrap().to_owned());
```

Comment:
I personally used to use semantically pedantically correct `to_owned()` in this situations, but since switched to `.to_string()`, as that is a tiny bit easier in terms of cognitive load. Both are fine though!